README.rst

```diff
@@ -1 +1,5 @@
 This is the Artefactual Storage Service for Archivematica.
+
+Start worker:
+
+celery -A storage_service worker -l info
```

Comment:
ansible role updated, merged to master branch  https://github.com/artefactual-labs/ansible-archivematica-src/pull/92